src/main/java/domain/result/Incomes.java

```diff
@@ -1,47 +1,57 @@
 package domain.result;
 
-import domain.Name;
+import domain.participant.Dealer;
+import domain.participant.Player;
 
+import java.util.LinkedHashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.Objects;
+import java.util.stream.Collectors;
 
 public class Incomes {
 
-    private final Map<Name, Income> incomes;
+    private final List<Player> players;
 
-    public Incomes(Map<Name, Income> incomes) {
-        this.incomes = incomes;
+    public Incomes(List<Player> players) {
+        this.players = players;
     }
 
-    public int getDealerIncome() {
-        int sumOfPlayerIncome = incomes.values()
-                .stream()
+    public Income dealerIncome(Dealer dealer) {
+        int sumOfPlayerIncome = players.stream()
+                .map(player -> player.determineIncome(dealer.decideStatus(player)))
                 .mapToInt(Income::getIncome)
                 .sum();
-        return -sumOfPlayerIncome;
+        return new Income(-sumOfPlayerIncome);
     }
 
-    public Map<Name, Income> getIncomes() {
-        return incomes;
+    public Map<Player, Income> playersIncomes(Dealer dealer) {
+        return players.stream()
+                .collect(Collectors.toMap(
+                        player -> player,
```

Comment:
자기 자신을 반환하는 경우를 위해 Funtion에선 identity()라는 정적 메서드를 제공합니다.
```suggestion
                        Funtion.identity(),
```

Reply:
player라는 것을 명시적으로 보여주는 게 좀 더 가독성이 있다고 생각해서 Function.identity()를 사용하지 않았는데,
정적 메서드를 쓰는 것을 추천하시는 이유가 있을까요?

Reply:
List의 of 메서드, Objects의 equals 메서드 등등처럼 가급적 SDK에서 제공하는 API를 사용하자는 방향입니다.
언급한 두개의 정적 메서드를 사용할 때 처럼 명백한 이득은 없지만요 ㅎ